src/printer.js

```diff
@@ -989,10 +989,7 @@ function genericPrintNoParens(path, options, print, args) {
       parts.push(opening);
 
       if (n.alternate) {
-        const hasBraces = isCurlyBracket(con);
-        const isEmpty = isEmptyBlock(con);
-
-        if (hasBraces && !isEmpty) {
```

Comment:
Note that we're no longer outputting

```
if (1) {}
else {}
```

but instead

```js
if (1) {
} else {
}
```

so the `isEmpty` condition isn't useful anymore